jwst/cube_skymatch/cube_skymatch_step.py

```diff
@@ -186,12 +184,12 @@ def _set_cube_bkg_meta(self, model, sky_cube):
         model.meta.background.subtract = self.subtract
         model.meta.background.level = None
         if np == 0:
-            meta.background.polynomial_info.append(pinfo)
+            model.meta.background.polynomial_info.append(pinfo)
         else:
-            meta.background.polynomial_info[0] = pinfo
+            model.meta.background.polynomial_info[0] = pinfo
 
     def _set_model2d_bkg_meta(self, model3d, model2d, channel):
-        _extend_2d_schema(model2d)
+        # _extend_2d_schema(model2d)
```

Comment:
Not sure commenting out `extend_2d_schema` is correct but it is not defined anywhere. Should it be imported from somewhere instead?

Reply:
Probably not. I was using it before things got hardcoded in the schema, see https://github.com/spacetelescope/jwst/pull/794